setup/resource_bigquery.tf

```diff
@@ -14,8 +14,17 @@ resource "time_sleep" "wait_for_bq_api" {
 }
 
 resource "google_bigquery_dataset" "four_keys" {
-  dataset_id = "four_keys"
-  location   = var.bigquery_region
+  dataset_id                 = "four_keys"
+  delete_contents_on_destroy = false
+  location                   = var.bigquery_region
+  access {
```

Comment:
access needed to be added and set. This was the main issue for inconsistent plan.